<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ROOPA" failures="0" tests="2" name="com.vtiger.contactsTest.CreateContactsTest" time="0.063" errors="1" timestamp="2022-05-31T00:57:21 GMT-07:00" skipped="1">
  <testcase classname="com.vtiger.contactsTest.CreateContactsTest" name="creatContactTest" time="0.000">
    <skipped/>
  </testcase> <!-- creatContactTest -->
  <system-out/>
  <testcase classname="com.vtiger.contactsTest.CreateContactsTest" name="beforesuit1Test" time="0.063">
    <error message=".\src\test\resources\CommonDta.Properties (The system cannot find the file specified)" type="java.io.FileNotFoundException">
      <![CDATA[java.io.FileNotFoundException: .\src\test\resources\CommonDta.Properties (The system cannot find the file specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:211)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:153)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:108)
at com.SDET34l1.genericUtility.FileLibrary.openPropertyFile(FileLibrary.java:25)
at com.SDET34l1.genericUtility.BaseClass.beforesuit1Test(BaseClass.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- beforesuit1Test -->
  <system-out/>
</testsuite> <!-- com.vtiger.contactsTest.CreateContactsTest -->
